source/fundamentals/time-series.txt

```diff
@@ -0,0 +1,100 @@
+.. _csharp-time-series:
+
+=======================
+Time Series Collections
+=======================
+
+.. default-domain:: mongodb
```

Comment:
S: you can remove this default-domain directive, we don't use it anymore (not sure what it does tbh)